### ISSUE TITLE
Let the user resize the window

### ISSUE THREAD

**Issue by ghost** [excessive/ludum-dare-37#3]
I don't know if the window is resizable during midgame, but since the mouse is captured, it doesn't really matter. However, it's resizable while not playing, and it's useful to resize it because, for example, setting fullscreen on and off leaves the window with the fullscreen size and it can't be shrunk again to normal. I didn't see any issues with doing so.

**Comment by shakesoda** [comment]
yeah, there's no reason we can't enable the flag, just need to test if anything breaks.

if you want to toggle fullscreen in-game you can use alt-enter.
